### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by eliwaksbaum**
allow the existing unit tests to be run on the current vagrant build, see #398

### CODE REVIEW

File: public/cantusdata/test/management/commands/test_import_all_data.py [hunk 1]
```diff
@@ -9,7 +9,8 @@ def setUp(self):
         pass
 
     def test_import_all_data(self):
-        call_command('import_data', 'test/test_chants.csv', '1', all=True)
+        call_command('import_data', 'concordances')
+        call_command('import_data', 'manuscripts')
```
Comment (napulen) [inline public/cantusdata/test/management/commands/test_import_all_data.py:13]: The original code seems to be importing chants, whereas the new one `concordances` and `manuscripts`. Is that the expected behaviour? That is, will the unit tests pass/fail accordingly?

File: public/cantusdata/helpers/expandr.py [hunk 2]
```diff
@@ -86,7 +86,9 @@ class PositionExpander(object):
     position_data_base = None
 
     def __init__(self):
-        self.csv_file = csv.DictReader(open("data_dumps/position_names.csv"))
+        cantusdata_path = os.path.split(os.path.dirname(__file__))[0]
+        public_path = os.path.split(cantusdata_path)[0]
+        self.csv_file = csv.DictReader(open(os.path.join(public_path, "data_dumps", "position_names.csv")))
```
Comment (napulen) [inline public/cantusdata/helpers/expandr.py:91]: If I understand this correctly (i.e., getting the absolute path of django project), this is available through the `settings` file of the project. You could import that path in the `settings.py` of the django project and avoid a redundant call to `os.path.dirname(__file__)`. 

Please check the `settings.py` and let us know if that's an appropriate import for this information.

**Review by napulen — COMMENTED** [review]
See comments.

File: public/cantusdata/test/management/commands/test_import_all_data.py (hunk 1 above)
Comment (eliwaksbaum) [reply public/cantusdata/test/management/commands/test_import_all_data.py:13]: The other tests are not using the data that is imported here. The point of this specific test is to test the ability to import data and as soon as it's done it deletes everything. I changed it from chants to concordances and manuscripts as you can't import chants independently anymore, nor does it seem like you can provide a directory as an argument to import stuff from.

File: public/cantusdata/helpers/expandr.py (hunk 2 above)
Comment (eliwaksbaum) [reply public/cantusdata/helpers/expandr.py:91]: yup. i'll make the fix

File: public/cantusdata/test/management/commands/test_import_all_data.py (hunk 1 above)
Comment (napulen) [reply public/cantusdata/test/management/commands/test_import_all_data.py:13]: Yes, you can't import chants independently anymore. If the purpose is importing data and deleting it right after, then this is perfect. Thanks!

**Comment by napulen** [comment]
Looking good now! Thanks!

**Review by napulen — APPROVED** [review]

**Pull request merged**